migrations/versions/131eb148fd57_add_task_table.py

```diff
@@ -0,0 +1,45 @@
+"""Add Task table
+
+Revision ID: 131eb148fd57
+Revises: 1eb0259b6faf
+Create Date: 2019-05-06 18:53:30.357780
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = '131eb148fd57'
+down_revision = '5279ea22b197'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
I don't think we need these kinds of comments in the code ;)

Reply:
Yeah sure, we don't need that. Didn't come up to remove this everywhere... but maybe it'd be better to apply that change to all migration scripts at once in next, smaller PR. Post a new issue with a screenshot of that, please.